invenio_app_ils/circulation/config.py

```diff
@@ -74,6 +74,8 @@
 ILS_CIRCULATION_NOTIFICATION_OVERDUE_REMINDER_INTERVAL = 3
 #: The maximum duration of a loan request
 ILS_CIRCULATION_LOAN_REQUEST_DURATION_DAYS = 60
+#: Loan Request Offset Value
+ILS_CIRCULATION_LOAD_REQUEST_OFFSET = 0
```

Comment:
```suggestion
ILS_CIRCULATION_LOAN_REQUEST_OFFSET = 0
```
Can you also please describe a little bit better what this config will do? `Offset` might not very clear to everyone.

Reply:
oops, I misspelled, thanks!

I will update the comment to describe it clearly, is this okay?
```
#: The minimum number of days after which a loan can start from the date of request
```